languagetool-http-client/src/main/java/org/languagetool/remote/SentenceAnnotator.java

```diff
@@ -334,12 +334,46 @@ private static void runAutomaticAnnotation(AnnotatorConfig cfg) throws IOExcepti
     cfg.out.close();
   }
 
+  // surround by double quotes and CSV-style escape of field-internal quotes
+  static private StringBuilder prepareFieldForCSV(String fieldValue) {
+    if (fieldValue.contains("\"") || fieldValue.contains(",")) {
+      return new StringBuilder()
+        .append("\"")
+        .append(fieldValue.replaceAll("\"", "\"\""))
+        .append("\"");
+    } else {
+      return new StringBuilder(fieldValue);
+    }
+  }
+
```

Comment:
Sometimes single quotes are also considered (just in case): 
```
public String escapeSpecialCharacters(String data) {
    String escapedData = data.replaceAll("\\R", " ");
    if (data.contains(",") || data.contains("\"") || data.contains("'")) {
        data = data.replace("\"", "\"\"");
        escapedData = "\"" + data + "\"";
    }
    return escapedData;
}
```

`\R` are line breaks. Probably not needed.

Reply:
Hm, right. I don't know how minimal quoting on pandas will work with unnecessary quotes (such as around `'`). But I don't think it'll hurt.

As for replacing the carriage returns, why whitespaces? Can't we just nuke them?

Reply:
Or, uh, just **force** everyone to use Unix-style line endings on **all** their repos **all the time**, because CRLF is a cancer..?